jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/handler/PodTemplateHandler.java

```diff
@@ -75,6 +79,16 @@ private List<Volume> getVolumes(ResourceConfig config) {
         return ret;
     }
 
+    public JobTemplateSpec getJobTemplate(ResourceConfig config, String restartPolicy, List<ImageConfiguration> images) {
+      return new JobTemplateSpecBuilder()
+          .withMetadata(createPodMetaData())
+          .withSpec(createJobSpec(config, restartPolicy, images))
+          .build();
+    }
 
-
+    private JobSpec createJobSpec(ResourceConfig config, String restartPolicy, List<ImageConfiguration> images) {
+      return new JobSpecBuilder()
+          .withTemplate(getPodTemplate(config, restartPolicy, images))
+          .build();
+    }
```

Comment:
Umm, this doesn't look like the right place for these methods. Do you think they can be shifted somewhere else?

Reply:
Would the `CronJobHandle` class be the right place for these methods?

Reply:
If it's used only by CronJobHandler, then it makes sense to move there there.